entities/communication/voiceCall.json

```diff
@@ -0,0 +1,70 @@
+{
+  "type": "object",
+  "properties": {
+    "uid": {
+      "type": "string",
+      "description": "Unique identifier for the call"
+    },
+    "duration": {
+      "type": "int",
+      "description": "Duration of the call in seconds"
+    },
+    "created_at": {
+      "type": "timestamp",
+      "description": "Time the call was created"
+    },
+    "updated_at": {
+      "type": "timestamp",
+      "description": "Time the call was last updated"
+    },
+    "status": {
+      "type": "enum",
+      "values": [
+        "incoming call",
+        "answered by staff",
+        "missed",
+        "answered by vonage",
+      ],
+      "description": "Status of the call"
+    },
+    "direction": {
+      "type": "enum",
+      "values": [
+        "inbound/outbound"
+      ],
+      "description": "Whether the call is inbound or outbound"
+    },
+    "staff_uid": {
+      "type": "string",
+      "description": "Unique identifier for the staff member who answered the call"
+    },
+    "client_uid": {
+      "type": "string",
+      "description": "Unique identifier for the client who made the call"
+    },
+    "from_number_id": {
```

Comment:
is this a uid? if yet, make it from_number_uid
I think you should use from_line_number_uid. Number is very generic. this is the line number. Apply the same to all properties with number